app/api/helpers/permissions.py

```diff
@@ -87,5 +87,113 @@ def decorated_function(*args, **kwargs):
 
     return decorated_function
 
+@second_order_decorator(jwt_required)
+def is_organizer(f):
+    """
+    Allows only Organizer to access the event resources.
+    :param f:
+    :return:
+    """
+    @wraps(f)
+    def decorated_function(*args, **kwargs):
+        user = current_identity
+        if kwargs['event_id'] and user.is_organizer(kwargs['event_id']):
```

Comment:
It will still give you a keyerror if not defined, use `if 'event_id' in kwargs` or `if kwargs.get('event_id') is not None`